src/main.rs

```diff
@@ -919,28 +922,52 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_function_call() {
+        parse_str(
+            r#"
+                fn test() {}
+                test()
+            "#,
+        );
+    }
+
+    #[test]
+    fn test_function_call_with_args() {
+        parse_str(
+            r#"
+                fn test(a int, b int) {}
+                test(1, 2)
+            "#
+        );
+    }
+
     #[test]
     fn test_variable() {
         parse_str(
             r#"
-                fn test() string {
-                    let foo = "1234"
-                }
+                let foo = "1234"
             "#,
         );
     }
 
     #[test]
-    fn test_variable_2() {
+    fn test_variables() {
         parse_str(
             r#"
-                fn test() string {
-                    let foo = ""
-                    let bar = foo
-                    let baz = bar
-                    return baz
-                }
+                let foo = ""
+                let bar = foo
+                let baz = bar
```

Comment:
See last comment this is a difficult thing to support.  
Maybe later on we can support this.  
For now i guess we can better not have this test.

Reply:
Removed ;-)